docs/topics/feed-exports.rst

```diff
@@ -258,6 +259,15 @@ which uses safe numeric encoding (``\uXXXX`` sequences) for historic reasons.
 
 Use ``utf-8`` if you want UTF-8 for JSON too.
 
+.. setting:: FEED_EXPORT_COMPRESSION
+
+FEED_EXPORT_COMPRESSION
+-----------------------
+
+Default: ``False``
+
+Set to ``True`` to enable GZip compression.
```

Comment:
:lipstick: Maybe we should call it ‘gzip’ and link to https://en.wikipedia.org/wiki/Gzip for reference.